.travis.yml

```diff
@@ -18,4 +18,4 @@ script:
   - npm run test
   - npm run jshint
 after_success:
-  - npm run coveralls
\ No newline at end of file
+  - npm run coveralls
```

Comment:
We can remove this entirely in another PR since we don't have a coveralls integration set up.